### PR TITLE
fix: encrypted content goes black in IE/Edge when video element focused

### LINKED ISSUE #6270: DRM content playback issue in IE11 and Edge
## Description
Starting with video.js version 7.5.5 on IE11 and edge, if you play DRM content using big-play-button, only sound comes out and the screen is black.
This problem is caused by the addition of a function to focus on the screen when big-play-button is clicked in 7.5.5.

7.5.5
```
if (this.mouseused_ && event.clientX && event.clientY) {
      silencePromise(playPromise);
      this.player_.tech(true).focus(); // Problem
      return;
    }
```

7.5.4
```
if (this.mouseused_ && event.clientX && event.clientY) {
      silencePromise(playPromise);
      // call handleFocus manually to get hotkeys working
      this.player_.handleFocus({}); // OK
      return;
    }
```

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Link to video js version 7.6.5.
2. Set DRM content to video js. ex) playready drm content
3. Click the Play Video button in the top left corner.

### Expected
DRM content play.

### Actual
Only sound comes out and the screen is black

### Error output
None

## Additional Information
Please include any additional information necessary here. Including the following:
#### videojs
7.5.5 or later versions
#### browsers
IE11 and Edge
#### OSes
windows 10

### PR DESCRIPTION
**Pull request by alex-barstow**
## Description
Fixes #6270. This appears to be caused by an unreported browser bug. It also reproduces with shaka player if you call `document.querySelector('video').focus()` while playing back DRM content. For now, the only solution is simply to avoid calling `videoEl.focus()` in IE/Edge for DRM. This unfortunately may prevent hotkeys from working correctly in IE/Edge.

## Specific Changes proposed
Modify the conditions under which we call `this.player_.tech(true).focus()` when the big play button is clicked.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -153,6 +153,7 @@
   },
   "vjsstandard": {
     "ignore": [
+      "build",
```
Comment (gkatsev) [inline package.json]: What type of errors were you seeing? I don't have any issues running the linter locally.

File: src/js/big-play-button.js [hunk 2]
```diff
@@ -46,8 +47,16 @@ class BigPlayButton extends Button {
 
     // exit early if clicked via the mouse
     if (this.mouseused_ && event.clientX && event.clientY) {
+      const sourceIsEncrypted = this.player_.usingPlugin('eme') &&
+                          this.player_.eme.sessions &&
```
Comment (gkatsev) [inline src/js/big-play-button.js]: should probably either line up or be indented once

Comment (alex-barstow) [reply src/js/big-play-button.js]: Yeah it was lined up in my editor but the indentation didn't carry over correctly for some reason. I'll figure it out.

File: package.json (hunk 1 above)
Comment (alex-barstow) [reply package.json]: My local `build/` doirectory was somehow outdated. This is not needed so I removed it.

**Comment by gkatsev** [comment]
Looks like *any* programmatic call to focus causes the video element to turn black.

**Pull request merged**